Resources/Prototypes/_Impstation/Entities/Objects/Misc/books_author.yml

```diff
@@ -58,7 +58,7 @@
 - type: entity
   parent: BookBase
   id: BookGatsby
-  name: The Great Gatsby by F. Scott Fitzgerald
+  name: the great gatsby by F. Scott Fitzgerald
```

Comment:
this should be capitalized, i think?

Reply:
proooobably, i was just going off the other books being uncapitalised

Reply:
all books should have capitalized titles. they r proper nowns